### ISSUE TITLE
use mean instead of sum for reconstruction error

### ISSUE THREAD

**Issue by poolio** [ganguli-lab/twpca#15]
currently we compute reconstruction error by summing over comopnents. Instead, we should compute the mean by normalizing by the number of elements. This will hopefully make the choice of regularization hyperparameters more robust across datasets.

**Comment by ahwillia** [comment]
This is addressed in https://github.com/ganguli-lab/twpca/pull/13

```python
tf.reduce_sum(self._mask * (self.pred - self._data)**2) / self.num_datapoints
```
